iot-hub/Samples/service/RoleBasedAuthenticationSample/Parameters.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.Azure.Devices;
+
+namespace RoleBasedAuthenticationSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'h',
+            "HostName",
+            Required = true,
+            HelpText = "The IoT hub host name.")]
```

Comment:
do you think an example would help here (my-iot-hub vs my-iot-hub.azure-devices.net)?

Reply:
yes it will